opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -777,3 +780,15 @@ def test_os_detector_solaris(self):
 
         self.assertEqual(resource.attributes[OS_TYPE], "solaris")
         self.assertEqual(resource.attributes[OS_VERSION], "666.4.0.15.0")
+
+
+class TestHostResourceDetector(unittest.TestCase):
```

Comment:
Please add a test analogous to `test_resource_detector_entry_points_os` so we can be sure that the entry point is hooked correctly

Reply:
Thanks for replay,  Added test_resource_detector_entry_points_host for unit test